sentry/processor.py

```diff
@@ -0,0 +1,232 @@
+""" raven.core.processors
+    ~~~~~~~~~~~~~~~~~~~~~
+:copyright: (c) 2010-2012 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
```

Comment:
url for AUTHORS and LICENSE  ?
Please describe why this copy paste. Are the (new) filtering options not good enough ?

Reply:
Sentry-sdk does not have a processor funcionality like raven, I've looked for a solution and have found comments about take processor class of raven and to do a custom implementation:

https://github.com/getsentry/sentry-python/issues/211#issuecomment-633101708
https://forum.sentry.io/t/how-to-customize-event-in-before-send/7457

By the way, I've changed this code.